operations/iuf/workflows/product_delivery.md

```diff
@@ -4,9 +4,9 @@ This section ensures the product content is loaded onto the system and available
 
 1. [Execute the IUF `process-media` and `pre-install-check` stages](#1-execute-the-iuf-process-media-and-pre-install-check-stages)
 1. [Update `customizations.yaml`](#2-update-customizationsyaml)
```

Comment:
Add Reference to 3rd sub section , Need to Update in 1.6 branch

# Product delivery

1. [Populate admin directory with files defining site preference](#3-populate-admin-directory-with-files-defining-site-preferences)

Reply:
Done the required changes